src/styles/colors.js

```diff
@@ -9,3 +9,18 @@ export const PRIMARY_WEAK = '#73dbff';
 export const PRIMARY_STRONG = '#17bcde';
 
 export const GREEN = '#b5ce4d';
+
+export const ARTICLES = {
```

Comment:
I meant those will be used only for articles but maybe there might be other usecases than this exercise. The idea behind using an object is that you will always use all those colors together I suppose.